Source/DafnyCore/Options/ICommandSpec.cs

```diff
@@ -38,6 +38,7 @@ public interface ICommandSpec {
     BoogieOptionBag.NoVerify,
     CommonOptionBag.EnforceDeterminism,
     CommonOptionBag.OptimizeErasableDatatypeWrapper,
+    CommonOptionBag.TestAssumptions
```

Comment:
I'm still slightly new the the modern options architecture we have here: will this option be available for `dafny translate`, too? I suspect it'll be used most often there, though also during `dafny run` and `dafny test`.

Reply:
Execution options are available for run,build,translate and test